tutorials/source-modeling/plot_fix_bem_in_blender.py

```diff
@@ -129,9 +138,68 @@
             op.join(bem_dir, 'inner_skull_orig.surf'))
 
 # Overwrite the original surface with the fixed version
+# In real study you should provide the correct metadata using ``volume_info=``
 mne.write_surface(op.join(bem_dir, 'inner_skull.surf'), coords, faces,
                   overwrite=True)
 
+###############################################################################
+# Editing the head surfaces
+# ^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Sometimes the head surfaces are faulty and require manual editing. We use
+# :func:`mne.write_head_bem` to convert the fixed surfaces to ``.fif`` files.
+#
+# Low-resolution head
+# ~~~~~~~~~~~~~~~~~~~
+#
+# For EEG forward modeling, it is possible that ``outer_skin.surf`` would be
+# manually edited. In that case, remember to save the fixed version of
+# ``-head.fif`` from the edited surface file for coregistration.
+
+# Load the fixed surface
+coords, faces = mne.read_surface(op.join(bem_dir, 'outer_skin.surf'))
+
+# Make sure we are in the correct directory
+if 'flash' in bem_dir:
```

Comment:
I wouldn't use this conditional, people can be expected to have up-to-date datasets. So just use the correct path for the current version of the dataset